src/jti_background.js

```diff
@@ -61,6 +64,30 @@ function saveTemplates (templateJSON, callback, responseData = null) {
     });
 }
 
+function getDomains (callback) {
+    var domainListCustom = {};
+    // Get the default domains
+    var domainList = $.map(DefaultDomainList, function (domain, index) {
+        domain.default = true;
+        return domain;
+    });
+    // Get the custom domains
+    chrome.storage.sync.get(StorageID, function (data) {
+        if (data[StorageID]) {
+            var domains = data[StorageID].options.domains;
```

Comment:
I don't think you need to declare this variable since we're not using it anywhere else.
Simply pass in `data[StorageID].options.domains` into the map

Reply:
:+1: